rune/libenclave/internal/runtime/pal/skeleton/Makefile

```diff
@@ -4,28 +4,25 @@ endif
 
 OUTPUT ?= ./
 SGX_SDK ?= /opt/intel/sgxsdk
-RA_TLS := ../../../../../../ra-tls
-LIBDIR := $(RA_TLS)/build/lib
-INCDIR := $(RA_TLS)/build/include
-RA_TLS_SERVER := $(RA_TLS)/ra-tls-server
+Enclave_Tls_Root ?= /opt/enclave-tls
+Enclave_Tls_Libdir := $(Enclave_Tls_Root)/lib
+Enclave_Tls_Incdir := $(Enclave_Tls_Root)/include
 TLS_SERVER ?=
 
 HOST_CFLAGS := -Wall -Werror -g -fPIC -z noexecstack \
 	       -Wno-unused-const-variable -std=gnu11 -I../kvmtool/include \
-	       -I$(SGX_SDK)/include -I$(INCDIR)
+	       -I$(SGX_SDK)/include -I$(Enclave_Tls_Incdir)
 ifdef TLS_SERVER
 HOST_CFLAGS += -DTLS_SERVER
 endif
 
 ENCL_CFLAGS := -Wall -Werror -static -nostdlib -nostartfiles -fPIC \
 	       -fno-stack-protector -mrdrnd -std=gnu11
-HOST_LDFLAGS := -fPIC -shared -Wl,-Bsymbolic
+HOST_LDFLAGS := -Wl,-start-group -fPIC -shared -Wl,-Bsymbolic,-rpath=$(Enclave_Tls_Libdir),-enable-new-dtags \
```

Comment:
Change to `-fPIC -shared -Wl,-Bsymbolic,-rpath=$(Enclave_Tls_Libdir),-enable-new-dtags -Wl,-start-group -lprotobuf-c -lutil -lbfd -lpthread -Wl,-end-group`